opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/exponential/buckets.py

```diff
@@ -0,0 +1,253 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+
+
+class BucketsBacking(ABC):
+    @abstractmethod
+    def size(self) -> int:
+        """
+        Returns the physical size of the backing array, which is
+        >= buckets.Len() the number allocated.
+        """
+
+    @abstractmethod
+    def grow_to(
+        self, new_size: int, old_positive_limit: int, new_positive_limit: int
+    ) -> None:
+        """
+        Grows the backing array into a new size and copies old entries into
+        their correct new positions.
+        """
+
+    @abstractmethod
+    def reverse(self, start: int, end: int) -> None:
+        """
+        Reverses the items in the backing array from [start, end[.
```

Comment:
```suggestion
        Reverses the items in the backing array from [start, end].
```

Reply:
Hm, what about [this](https://github.com/lightstep/otel-launcher-go/blob/main/lightstep/sdk/metric/aggregator/histogram/structure/exponential.go#L128)?

Reply:
UPDATED :)